### PR TITLE
switched to use correct unpkg domain for zxing lib

### PR DESCRIPTION
**Pull request by HeikoWitthoeft**
switched to use correct url for xzing lib for the live demo
https://unpkg.com/@zxing/library@latest/umd/index.min.js

### CODE REVIEW

File: docs/examples/qr-video/index.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h1 class="title">Scan QR Code from Video File</h1>
 
   </main>
 
-  <script type="text/javascript" src="https://unpkg.com/@zxing/library@latest"></script>
+  <script type="text/javascript" src="https://unpkg.com/@zxing/library@0.18.6/umd/index.min.js"></script>
```
[the remark this answers is not in the document]
Comment (rvalitov) [reply docs/examples/qr-video/index.html:65]: #483 uses `@latest` in all files. So I think it's not required to use a specific version.

[the remark this answers is not in the document]
Comment (rvalitov) [reply docs/examples/qr-video/index.html:65]: #483 uses @latest in all files. So I think it's not required to use a specific version.

**Review by rvalitov — CHANGES_REQUESTED** [review]

**Comment by WIStudent** [comment]
Latest should work again in version 0.19 (see https://github.com/zxing-js/library/pull/452) but unfortunately some broken tests prevented v0.19 from being pushed to npm.

**Comment by rvalitov** [comment]
@WIStudent Thank you so much for reporting that. Can you please let me know if you could fix the related tests?

**Comment by WIStudent** [comment]
@rvalitov Looking at the history of the "Continuous Integration" workflow it seems that the E2E test never ran successfully on github actions. https://github.com/zxing-js/library/actions/workflows/ci.yml

Back in summer I was researching some ways to scan QR codes in web, found @zxing/library, noticed that the demo page was broken, and created PR https://github.com/zxing-js/library/pull/452 to fix unpkg. But in the end I decided against using @zxing/library in my project. Just by coincidence I noticed now that the demo page is still broken despite https://github.com/zxing-js/library/pull/452 being merged.

So someone else more familiar with the codebase needs to look why the E2E tests are failing.

**Comment by rvalitov** [comment]
@WIStudent thank you for you prompt response. I'm curious what package did you use in your project instead of this library?

**Comment by WIStudent** [comment]
@rvalitov I was looking for a way to scan QR codes inside a [capactior](https://capacitorjs.com/) app and ended up using [@capacitor-community/barcode-scanner](https://www.npmjs.com/package/@capacitor-community/barcode-scanner) which does the scanning and processing in the native android/iOS part instead of the web part.

**Comment by rvalitov** [comment]
@WIStudent Ok, got it, thank you!

**Comment by rvalitov** [comment]
@werthdavid can you please look into this problem that library ver.0.19 is missing at npm? Perhaps, you could publish it there manually? I'm not familiar with the tests, but perhaps you could give some tips if it's easy to fix them or not? Or if the failing tests are really required or perhaps that's something old that could be deleted?

**Comment by werthdavid** [comment]
@rvalitov it didn't publish to npm because the tests failed. I started commenting tests out on my machine but its really a lot I have to disable so the tests won't fail... not sure if that's a good idea imho. on the other hand the versions minor to 0.19 are not really working as well either. What do you mean? publish without tests?

**Comment by rvalitov** [comment]
@werthdavid Yes, I suggest to publish without tests. At least this gives more freedom to end users and they can use the version they need by pinning to a specific version. So in case the version 0.19 is not good for someone, then they can stay with version 0.18. But now if a user needs version 0.19 then packaging it manually without using npm is a little more troublesome. So, a released 0.19 version in npm would help a lot.

**Comment by werthdavid** [comment]
I hope v0.19.1 works

**Comment by rvalitov** [comment]
@werthdavid Thank you! FYI, unfortunately I couldn't use and test the v0.19.1, because both the `ngx-scanner` and `browser` require and use `^0.18.5`, therefore the `npm` does not allow me to use v.0.19.1

**Comment by werthdavid** [comment]
I can provide you a tgz file for testing if you want

**Comment by werthdavid** [comment]
Should be fixed. Reopen if not

**Pull request closed without merging**